Contest_API.md

```diff
@@ -142,7 +142,13 @@ their expected behavior, if implemented.
 
 Standard [HTTP status
 codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) are
-returned to indicate success or failure.
+returned to indicate success or failure. When there is a failure (status codes 4xx or 5xx), the
+response message body must include a JSON element that contains the attributes 'code' (a number, identical to the HTTP status code returned) and 'message' (a string) with further information suitable for the user making the request, as per the following example:
+
+```json
+{"code":400,
+ "message":"Teams cannot send clarifications to another team"}
```

Comment:
This is not a *great* example, because the code is arguably wrong. I would say that trying to send a message to a team when authenticated as a team would be a `403 Forbidden`.

Reply:
Example updated to be a 403. I also noticed the clarification POST says you must return a "400 error code" so I changed this to a "4xx HTTP status code".